src/py42/services/storage/archive.py

```diff
@@ -109,38 +113,31 @@ def create_restore_session(
 
     def start_restore(
         self,
-        guid,
+        device_guid,
         web_restore_session_id,
-        path_set,
+        restore_groups,
         num_files,
         num_dirs,
-        size,
-        zip_result=None,
+        num_bytes,
         expire_job=None,
         show_deleted=None,
         restore_full_path=None,
-        timestamp=None,
-        backup_set_id=None,
+        restore_to_server=None,
     ):
-        """Submits a web restore job.
-        See https://console.us.code42.com/apidocviewer/#WebRestoreJob-post
-        """
-        uri = u"/api/WebRestoreJob"
+        """Submits a web restore job."""
+        uri = u"/api/v9/restore/web"
```

Comment:
can we add test for the change.

Reply:
There is a test already that makes sure the URL is correct.